recipes/readline/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+./configure --prefix=$PREFIX
```

Comment:
Maybe add an `|| { cat config.log; exit 1; }` at the end of this line, in case configuring fails.